src/PProf.jl

```diff
@@ -217,8 +217,35 @@ function pprof(data::Union{Nothing, Vector{UInt}} = nothing,
         writeproto(io, prof)
     end
 
-    if web run(`$go_pprof -http=$webhost:$webport $out`) end
+    if web
+        if !isnothing(PProf.pprof_proc[1])
+            kill(PProf.pprof_proc[1])
+        end
+
+        PProf.pprof_proc[1] = run(`$go_pprof -http=$webhost:$webport $out`, wait = false)
+    end
+
     out
 end
 
+const pprof_proc = Union{Base.Process, Nothing}[nothing]
+
+export @pprof
+"""
+    @pprof expr
+
+Alias for
+```julia
+@profile expr
+pprof()
+```
+"""
+macro pprof(ex)
+    quote
+        Profile.clear()
```

Comment:
I would re-export clear from `PProf` and drop it from this macro. It often can be useful to accumulate more data and so the `clear` should be conscious.

Reply:
clear is not exported by `Profile` so I will just import it, this way you can just call `PProf.clear()`, the same way you would call `Profile.clear()`